WcaOnRails/app/jobs/dump_developer_database.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class DumpDeveloperDatabase < ApplicationJob
+  queue_as :default
+
+  def perform
+    # Create developer database dump every 3 days.
+    last_developer_db_dump = Timestamp.find_or_create_by(name: 'last_developer_db_dump')
+    if last_developer_db_dump.date.nil? || last_developer_db_dump.date < 3.days.ago
```

Comment:
@larspetrus I think the first approach is not gonna work, as it would try to find a timestamp with `date: 4.days.ago`, and when trying the second I got an error that 0 is not a valid value for DATETIME, which makes perfect sense.
I came up with yet another idea, please let me know what do you think about this =)